source/backend/api/Areas/Projects/Controllers/ProjectController.cs

```diff
@@ -40,6 +40,28 @@ public ProjectController(IProjectService projectService, IMapper mapper)
             _mapper = mapper;
         }
 
+        /// <summary>
+        /// Get the specified Project by Id.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("{id}")]
+        [HasPermission(Permissions.ProjectView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(ProjectModel), 200)]
+        [ProducesResponseType(typeof(ProjectModel), 404)]
+        [SwaggerOperation(Tags = new[] { "project" })]
+        public async Task<IActionResult> GetById([FromRoute] long id)
+        {
+            var project = await _projectService.GetById(id);
+            if (project is null)
+            {
+                return NotFound();
+            }
+
+            return Ok(_mapper.Map<ProjectModel>(project));
```

Comment:
Just double-checking that calling the method **Ok()** here is equivalent to returning **JsonResult** ?

Reply:
According to this [link](https://learn.microsoft.com/en-us/aspnet/core/web-api/advanced/formatting?view=aspnetcore-7.0) there is a subtle difference in returning Ok in that the client can negotiate the return type via the **Accept** header in the request. I'm not opposed to start using Ok but would like that we all agree in a common pattern as I see a mix of Ok and JsonResult even in this same file

Reply:
I think we should be making an explicit response for the Api, not the content type.